src/Field/SurveyLikertField.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace GFExcel\Field;
+
+/**
+ * A field transformer for the `likert` field from the Gravity Forms Survey Plugin.
+ * @since $ver$
+ * @see \GF_Field_Likert
+ */
+class SurveyLikertField extends SeparableField {
+	/**
+	 * @inheritdoc
+	 * @var \GF_Field_Likert
+	 */
+	protected $field;
+
+	/**
+	 * @inheritdoc
+	 * @since $ver$
+	 */
+	public function getFieldValue( $entry, $input_id = '' ) {
+		if ( ! $this->useScoreOutput() ) {
+			return parent::getFieldValue( $entry, $input_id );
+		}
+
+		$value = $entry[ $input_id ?: $this->field->id ] ?? ':';
```

Comment:
Instead of compact, let's shoot for simple notation. Please break into multiple lines.